daml_finance_dep.bzl

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+quickstart = {
+    "version": "9126428a130336c62a7231fc002b0632a2f4c235",
```

Comment:
Just fyi, this commit includes the new `Holding` tutorial script from @markus-da

Reply:
Yes, the change is included in this commit.